draft-irtf-cfrg-vdaf.md

```diff
@@ -414,13 +414,13 @@ security considerations for DAFs and VDAFs.
 * Define Prio3MultihotCountVec, a variant of Prio3 for aggregating bit vectors
   with bounded weight.
 
-* FLP: Allow the output of the circuit to be a vector. This makes it possible
+* FLP: allow the output of the circuit to be a vector. This makes it possible
```

Comment:
I lean towards leaving these occurrences in the changelog as-is, because I view the sentence as starting with "Allow", for example, and the "FLP:" that comes before it as a label, outside of the sentence structure.

Reply:
Done.